### DISCUSSION TITLE
Subtitles to Week 7 dialoges

### DISCUSSION THREAD

**Discussion by Windows1Door2Daniel** [FunkinCrew/Funkin#1674] [Ideas]
Just that

**Comment by ghost** [comment]
this would be cool for the people that cant he- oh wait this is a game based around hearing

**Comment by objectionist** [comment]
> 
> 
> this would be cool for the people that cant he- oh wait this is a game based around hearing

i mean, its still a good idea. subtitles for spoken dialogue helps a lot even if you aren't deaf, sometimes people pronounce things in ways that can't be well-heard

**Comment by ghost** [comment]
> > > this would be cool for the people that cant he- oh wait this is a game based around hearing
> > 
> > 
> > i mean, its still a good idea. subtitles for spoken dialogue helps a lot even if you aren't deaf, sometimes people pronounce things in ways that can't be well-heard
> 
> And as a guy with Spanish as their native language... I can barely understand lines in week 7.

AHAHAHAHAHAHAHAHAHAHAHAHA `Change "Dialog" To 100% "English"`

**Comment by ghost** [comment]
> > > > this would be cool for the people that cant he- oh wait this is a game based around hearing
> > > 
> > > 
> > > i mean, its still a good idea. subtitles for spoken dialogue helps a lot even if you aren't deaf, sometimes people pronounce things in ways that can't be well-heard
> > 
> > 
> > And as a guy with Spanish as their native language... I can barely understand lines in week 7.
> 
> AHAHAHAHAHAHAHAHAHAHAHAHA `Change "Dialog" To 100% "English"`

shut up

**Comment by ghost** [comment]
> > > > this would be cool for the people that cant he- oh wait this is a game based around hearing
> 
> > > 
> 
> > > 
> 
> > > i mean, its still a good idea. subtitles for spoken dialogue helps a lot even if you aren't deaf, sometimes people pronounce things in ways that can't be well-heard
> 
> > 
> 
> > And as a guy with Spanish as their native language... I can barely understand lines in week 7.
> 
> 
> 
> AHAHAHAHAHAHAHAHAHAHAHAHA `Change "Dialog" To 100% "English"`

how about you `Change "Silence TimeGuy" to 420% "Shut Up"`

**Comment by ninjamuffin99** [comment]
will prob happen with 'translations' and whatnot from kickstartr stretch goal

**Comment by GOOBOZA** [comment]
thats not a god idea because it will just be an dialogue basically with a voicing but it will be a good idea if you are turn it on the settings.

**Comment by Windows1Door2Daniel** [comment]
I'm deaf

**Reply by Windows1Door2Daniel** [reply]
I cant hear things well

And i have problems understanding words
